examples/TypeScriptMessaging/package.json

```diff
@@ -28,7 +28,7 @@
     "react-native-image-crop-picker": "^0.39.0",
     "react-native-image-resizer": "^1.4.5",
     "react-native-quick-sqlite": "^8.0.2",
-    "react-native-reanimated": "^3.1.0",
+    "react-native-reanimated": "~3.1.0",
```

Comment:
Done because when using `^` the version in yarn.lock is 3.2.0 and it throws a babel error:

```
 ERROR  Error: Failed to create a worklet. Did you forget to add Reanimated Babel plugin in babel.config.js? See installation docs at https://docs.swmansion.com/react-native-reanimated/docs/fundamentals/installation#babel-plugin., js engine: hermes
 LOG  Running "SampleApp" with {"rootTag":1,"initialProps":{}}
 ERROR  Invariant Violation: "SampleApp" has not been registered. This can happen if:
* Metro (the local dev server) is run from the wrong folder. Check if Metro is running, stop it and restart it in the current project.
```